spec/models/user_spec.rb

```diff
@@ -15,5 +15,80 @@
 require 'rails_helper'
 
 RSpec.describe User, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe 'バリデーション' do
+    subject(:user) do
+      described_class.new(email: email, password: password, password_confirmation: password_confirmation)
```

Comment:
省略で記載できるかと！他の部分も同様です。
```suggestion
      described_class.new(email: , password: , password_confirmation: )
```